sparkviz/__init__.py

```diff
@@ -53,9 +50,8 @@ def __init__(self, pdf, sdf, sampled):
     self.sdf = sdf
     self.sampled = sampled
     from IPython.display import display
-    # TODO(communicate if we've sampled data to the front end?)
-    self.display = display(pdf, display_id=True)
-
-  # TODO(support call backs and push down others)
+    # TODO(do something where we have more control besides pandas directly)
+    with pd.option_context('display.html.table_schema', True):
+      self.display = display(pdf, metadata={"application/json": { "sampled": sampled} }, display_id=True)
```

Comment:
You'll need the key to be `application/vnd.dataresource+json` instead of plain json here